_glossary/workload-principal.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Workload Principal
+type: markdown
+---
+Identifies the verifiable authority under which a **Workload** runs.
+Istio service-to-service authentication is used to produce the **Workload Principal**.
+By default **Workload Principals** are compliant with the SPIFFE ID format.
+  * Multiple **Workloads** may share the same **Workload Principal**, but each **Workload** has a single canonical **Workload Principal**.
+  * **Worklooad Principals** are accessible in Istio configuration as the `source.user` and `destination.user` attributes.
```

Comment:
Shame we can be consistent here with "user" vs "principal".

Reply:
Also typo.. Worklooad (double o)

Reply:
Yeah we want to change this in an upcoming release but didn't want to do it right before 0.2.

Plan is to rename to source.principal and destination.principal, and we'll also have request.auth.principal for auth credentials.

Reply:
Fixed typo.